src/wwwroot/services/auth.js

```diff
@@ -251,7 +251,8 @@
           daily: mapLimit(response.data.daily),
           hourly: mapLimit(response.data.hourly),
           hasLimits: !response.data.noLimits,
-          endDate: mapDate(response.data.endDate)
+          endDate: mapDate(response.data.endDate),
+          hasDkimReady: response.data.dkimConfigurationReady          
```

Comment:
The name of the field is `dkimConfigurationRequired`.

You can fix it inverting the logic, something like:

```
    hasDkimReady: !response.data.dkimConfigurationRequired
```

But I think that it is better to keep the same meaning, for example:


```
    requiresDkimConfiguration: !!response.data.dkimConfigurationRequired
```